cypress/e2e/SignIn.cy.ts

```diff
@@ -10,7 +10,7 @@ describe('Name and Email Validation', () => {
     cy.visit(SIGN_IN_PATH);
     // Signing in with a wrong email format
     cy.signInAndCheck(WRONG_EMAIL);
-    // Siging in with a valid email
+    // Signing in with a valid email
     cy.signInAndCheck(GRAASP);
```

Comment:
Because we have password now, what about specifying the auth method here ?

```suggestion
    cy.signInByMailAndCheck(GRAASP); // Or a better name
```
```suggestion
    cy.signInAndCheck(GRAASP);
```